Update FAQ page (#6948)
circleci/circleci-docs: jekyll/_cci2/faq.md

> For an introduction to execution environments and images, see the [Executors and Images guide]({{site.baseurl}}/2.0/executor-intro/).

Reviewer: ```suggestion
For an introduction to execution environments and images, see the [Introduction to Execution Environments]({{site.baseurl}}/2.0/executor-intro/).
```

{:.no_toc}
Coordinated Universal Time (UTC) is the time zone in which schedules are interpreted.

### Are scheduled pipelines guaranteed to run at precisely the time scheduled?
{: #are-scheduled-pipelines-guaranteed-to-run-at-precisely-the-time-scheduled }
{:.no_toc}
CircleCI provides no guarantees about precision. A schedule will be run as if the commit was pushed at the configured time.

---

## Workflows
{: #workflows }

### How many jobs can I run concurrently?
{: #how-many-jobs-can-i-run-concurrently }
{:.no_toc}
The number of jobs you can run concurrently differs between [plans](https://circleci.com/pricing/). When using workflows to schedule jobs, you can use a [fan-out/fan-in method]({{site.baseurl}}/2.0/workflows/#fan-outfan-in-workflow-example) to run jobs concurrently.

### Can I use multiple executor types in the same workflow?
{: #can-i- use-multiple-executor-types-in-the-same-workflow }
{:.no_toc}
Yes, this is supported. See the [Sample Configuration]({{site.baseurl}}/2.0/sample-config/#sample-configuration-with-multiple-executor-types) page for examples.

### Can I build only the jobs that changed?
{: #can-i-build-only-the-jobs-that-changed }
{:.no_toc}

> You can set up your workflows to conditionally run jobs based on specific updates to your repository. You can do this with [conditional workflows]({{site.baseurl}}/2.0/pipeline-variables/#conditional-workflows) and [dynamic configurations]({{site.baseurl}}/2.0/dynamic-config/). Dynamic configurations will dynamically generate CircleCI configuration and pipeline parameters, and run the resulting work within the same pipeline. You can read more about setting up this feature in our [API docs](https://github.com/CircleCI-Public/api-preview-docs/blob/path-filtering/docs/path-filtering.md#setup-workflows).

Reviewer: ```suggestion
You can set up your workflows to conditionally run jobs based on specific updates to your repository. You can do this with [conditional workflows]({{site.baseurl}}/2.0/pipeline-variables/#conditional-workflows) and [dynamic configurations]({{site.baseurl}}/2.0/dynamic-config/). Dynamic configurations will dynamically generate CircleCI configuration and pipeline parameters, and run the resulting work within the same pipeline.
```

### What is the best practice for specifying image versions?
{: #what-is-the-best-practice-for-specifying-image-versions }
{:.no_toc}

> It is best practice **not** to use the `latest` tag for specifying image versions. It is also best practice to use a specific version and tag, for example `circleci/ruby:2.4-jessie-node`, to pin down the image and prevent upstream changes to your containers when the underlying base distro changes. For example, specifying only `circleci/ruby:2.4` could result in unexpected changes from `jessie` to `stretch`. For more context, refer to [Docker image best practices]({{site.baseurl}}/2.0/using-docker/#docker-image-best-practices), and [CircleCI image best practices]({{site.baseurl}}/2.0/circleci-images/#best-practices).

Reviewer: ```suggestion
It is best practice **not** to use the `latest` tag for specifying image versions. It is also best practice to use a specific version and tag, for example `cimg/ruby:3.0-node`, to pin down the image and prevent upstream changes to your containers when the underlying base distro changes. For example, specifying only `cimg/ruby:3.0` could result in unexpected changes from `jessie` to `stretch`. For more context, refer to [Docker image best practices]({{site.baseurl}}/2.0/using-docker/#docker-image-best-practices), and [CircleCI image best practices]({{site.baseurl}}/2.0/circleci-images/#best-practices).
```

### What is the best practice for specifying image versions?
{: #what-is-the-best-practice-for-specifying-image-versions }
{:.no_toc}

> It is best practice **not** to use the `latest` tag for specifying image versions. It is also best practice to use a specific version and tag, for example `circleci/ruby:3.0.4-browsers`, to pin down the image and prevent upstream changes to your containers when the underlying base distribution changes. For example, specifying only `circleci/ruby:3.0.4` could result in unexpected changes from `browsers` to `node`. For more context, refer to [Docker image best practices]({{site.baseurl}}/2.0/using-docker/#docker-image-best-practices), and [CircleCI image best practices]({{site.baseurl}}/2.0/circleci-images/#best-practices).

Reviewer: ```suggestion
It is best practice **not** to use the `latest` tag for specifying image versions. It is also best practice to use a specific version and tag, for example `cimg/ruby:3.0.4-browsers`, to pin down the image and prevent upstream changes to your containers when the underlying base distribution changes. For example, specifying only `cimg/ruby:3.0.4` could result in unexpected changes from `browsers` to `node`. For more context, refer to [Docker image best practices]({{site.baseurl}}/2.0/using-docker/#docker-image-best-practices), and [CircleCI image best practices]({{site.baseurl}}/2.0/circleci-images/#best-practices).
```
